maps/CdaToBundle.map

```diff
@@ -550,7 +557,7 @@ group CdaPatientRoleToFhirPatient(source cda_patientRole : PatientRole, target f
     cda_patient.birthTime as cda_patient_birthTime where (cda_patient.birthTime.value.length() > 10) -> fhir_patient.birthDate as fhir_patient_birthDate,
       fhir_patient_birthDate.extension as extension,
       extension.url = 'http://hl7.org/fhir/StructureDefinition/patient-birthTime',
-      extension.value = (cda_patient_birthTime.value) "CdaPatientBirthTimeToFhirPatientBirthTimeExtension";
+      extension.value = (cda_patient_birthTime.value.toDateTime()) "CdaPatientBirthTimeToFhirPatientBirthTimeExtension";
```

Comment:
```suggestion
      extension.value = create('dateTime') as fhir_patient_birthTime_dateTime then TSDateTime(cda_patient_birthTime, fhir_patient_birthTime_dateTime) "CdaPatientBirthTimeToFhirPatientBirthTimeExtension";
```
Could you try this with MaLaC-HD? In Matchbox `toDateTime` is not supported.

Reply:
Works fine with MaLaC-HD